tests/test_flake8_trio.py

```diff
@@ -125,8 +125,22 @@ def test_eval(test: str, path: str):
     assert_expected_errors(plugin, include, *expected)
 
 
-# codes that should never error when run on sync code
-sync_errors = ["TRIO102", "TRIO107", "TRIO108", "TRIO109", "TRIO110"]
+# codes that can error when run on sync code, and should be excluded from
+# the SyncTransformer check. Expand this list when adding new checks if they
+# should also warn on sync.
+sync_errors_ignore = {
```

Comment:
well the names are clearly bad cause you're interpreting the function of this list completely opposite of how it should work :sweat_smile: Previously the list was which codes that were *not* allowed to be in the synctransformed code - which was problematic because it defaulted to not caring about new codes unless you explicitly remembered to add it to the list, now the list is which codes are the ones allowed - and so when this test raises an error you either have a bug where it's giving errors on sync code despite not being supposed to - or it should give errors on sync code too in which case you update the list.

So yes, 109 is checkable and *is* checked in sync code using the SyncTransformer interface

Changed the name to the super verbose `error_codes_ignored_when_checking_transformed_sync_code` and improved the comment. I tried a couple different ones previously and didn't find a good set (also it doesn't really need to be two statements, but explaining the functionality when I merged them into a one-liner got *super* confusing), so I like going the overly verbose and unambigious route.